index.bs

```diff
@@ -10331,25 +10331,27 @@ Note: Remember that interface objects for callback interfaces only exist if they
 when they do exist, they are not [=function objects=].
 
 
-<h5 id="es-interface-call">Interface object \[[Call]] method</h5>
+<h5 id="es-constructible-interfaces" oldids="es-interface-call">Constructible Interfaces</h5>
 
-If the [=interface=] is declared with a
-[{{Constructor}}] [=extended attribute=],
-then the [=interface object=]
-can be called as a function to create an object that implements that
-interface.  Interfaces that do not have a constructor will throw
-an exception when called as a function.
+If the [=interface=] is declared with a [{{Constructor}}] [=extended attribute=],
+then the [=interface object=] can be called as a constructor
+to create an object that implements that interface.
+Calling that interface as a function will throw an exception.
 
-The internal \[[Call]] method
-of the interface object behaves as follows, assuming
-|arg|<sub>0..|n|−1</sub> is the list
-of argument values passed to the constructor, and |I|
-is the [=interface=]:
+Interfaces that do not have a constructor will throw
+an exception when called.
+
+When evaluating the [=function object=] |F|,
+which is the interface object for a given non-callback [=interface=] |I|,
+assuming |arg|<sub>0..|n|−1</sub> as the list of argument values passed |F|,
+and |newTarget| as the NewTarget value, the following steps must be taken:
 
 <ol class="algorithm">
     1.  If |I| was not declared with a [{{Constructor}}]
         [=extended attribute=], then
         <a lt="es throw">throw a <emu-val>TypeError</emu-val></a>.
+    1.  If |newTarget| is <emu-val>undefined</emu-val>, then
```

Comment:
In https://github.com/tc39/ecma262/issues/529#issuecomment-207459518, @allenwb writes: 

>  "NewTarget" and "this value" should only be used in the top-level algorithms that defined built-ins. They should not be used in other abstract operations (because paragraph 3 of 9.3 doesn't generally apply to abstract operations). If an abstract operation that is used to factor the definition of one or more built-ins need to access those values they should be explicitly passed as parameters to the abstract operation (for example see https://tc39.github.io/ecma262/#sec-regexpalloc).

Aren't we in the case described by that last sentence, here?